tests/baselines/reference/asyncArrowFunction7_es5.js

```diff
@@ -9,6 +9,7 @@ var bar = async (): Promise<void> => {
 var _this = this;
 var bar = function () { return __awaiter(_this, void 0, void 0, function () {
     var _this = this;
+    // 'await' here is an identifier, and not an await expression.

```

Comment:
This is the reason we didn't do this before. This comment is now completely divorced from its context and makes no sense here.